tox.ini

```diff
@@ -11,24 +10,22 @@ envlist =
 passenv = PYTHONPATH DJANGO_SETTINGS_MODULE
 deps =
     django111: Django>=1.11,<2.0
-    django20: Django>=2.0,<2.1
-    django21: Django>=2.1,<2.2
     django22: Django>=2.2,<2.3
     django30: Django>=3.0,<3.1
     djangomaster: https://github.com/django/django/archive/master.tar.gz
+commands =
+    django-admin test --noinput
+
+[testenv:lint]
+deps =
     check-manifest
     flake8
     readme_renderer
-    pytest-runner
-    pytest
-
 commands =
     check-manifest --ignore tox.ini
     {envpython} setup.py check -m -r -s
     flake8 .
-    django-admin test --noinput
+skip_install = true
```

Comment:
We don't actually need to install the django-ecsmanage package and dependencies in order to lint it. This saves some time during this phase of the build. (azavea/django-amazon-ses#61)